changelogs/unreleased/1071-deprecation-warning-regarding-asyncio-mode-in-module-set-tests.yml

```diff
@@ -0,0 +1,6 @@
+description: Remove pytest.ini and move its content to pyproject.toml
+issue-nr: 1071
+change-type: patch
```

Comment:
You need to set `issue-repo` to `irt` as well, because the repo of the PR doesn't match the repo of the issue